pages/main.scss

```diff
@@ -131,3 +177,13 @@ p {
     }
   }
 }
+
+@media (min-width: map-get($grid-breakpoints, md)) {
```

Comment:
Let's use the bootstrap http://v4-alpha.getbootstrap.com/layout/overview/#responsive-breakpoints